docs/app/(docs)/getting-started/page.mdx

```diff
@@ -63,6 +69,14 @@ Now we need to invoke the TypeScript compiler (`tsc`) to build the bot and gener
 npm run build
 ```
 
+Or if using bun, you can run the following command to build the bot:
+
+```bash
+bun run build
+```
+
+If that fails, don't worry. You can get a pre-built version or run the bot directly without building, you just might encounter some errors with a few commands. If this does happen, you can get help in the [Discord Server](https://discord.gg/892GWhTzgs).
```

Comment:
> If that fails, don't worry. You can get a pre-built version or run the bot directly without building, you just might encounter some errors with a few commands. If this does happen, you can get help in the [Discord Server](https://discord.gg/892GWhTzgs).
>
> If it fails at building because you don't have enough resources to do so, or if it's for another reason, you can find pre-built versions [here](github.com/onesoft-sudo/sudobot/releases/latest) for linux and darwin.

I kinda feel like this explicitly states "You will encounter errors", which might not sound really good if you know what I mean. 
There are complexities here. First of all, local build is always preferred, so that the build is compatible with the installed Node.js versions. I'd say you should explicitly say that here. 
Something like this could be nice:

```
If you don't have enough resources, this command will fail with heap allocation errors. If that happens, or if you don't want to build it yourself, don't worry. You can download prebuilt versions for every release. The builds are tested on Node.js **v21**, however they should also work with **v20**.
You might see that only Linux and macOS (darwin) releases are available. This doesn't mean you cannot run the bot on Windows systems - only the native bindings are platform dependent. You don't need to worry about that in most cases and the bot will just work fine.

As always if you ever encounter errors with commands or you see something is not working as you expect, you can join our [Discord Server](https://discord.gg/892GWhTzgs) and ask for help!
```

What do you think?

Reply:
Done, check newest commit on it.